packages/api/lib/rulesHelpers.js

```diff
@@ -608,8 +609,10 @@ async function updateRuleTrigger(ruleItem, updates, knex) {
   case 'sqs':
     clonedRuleItem = await validateAndUpdateSqsRule(mergedRule);
     break;
-  default:
+  case 'onetime':
     break;
+  default:
+    throw new ValidationError(`Rule type \'${mergedRule.rule.type}\' not supported.`);
```

Comment:
I made these changes and added `onetime` as a case because of this unit test I wrote to check that PUT errors when a rule type is invalid [`68155b3` (#2871)](https://github.com/nasa/cumulus/pull/2871/commits/68155b3331587a648cfa8a71f3b863c9bfd173fb#diff-49139a123843cc0118f2bc2b3275724314181f1de87be662e46c5125aca6c844R973) 

I can add `scheduled` as a rule type if we want to keep this functionality. Otherwise, I can remove the unit test if we shouldn't be checking for valid rule types in `updateRuleTrigger`